package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "@openzeppelin/test-environment",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
Can you remove the version bump? We will take care of it when releasing.

Reply:
Removed the new version, thanks.